### PR TITLE
 upgrade to ember octane

### PR DESCRIPTION
**Pull request by mcfiredrill**

### CODE REVIEW

**Comment by mcfiredrill** [comment]
My understanding is that we can opt it to the rest of the ember octane features incrementally (native classes, glimmer components, etc). So as long as this isn't breaking anything I will merge it.

**Comment by sharpshark28** [comment]
> My understanding is that we can opt it to the rest of the ember octane features incrementally ([native classes](https://github.com/datafruits/datafruits/pull/272), glimmer components, etc). So as long as this isn't breaking anything I will merge it.

That's 💯 correct. It's worth briefly QA'ing to be sure all pages seem to load etc but you should be good to go.

File: app/templates/home/chat.hbs [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {{#if fastboot.isFastBoot}}
   <div class="main-content">
     <div class="loading-container">
-      <img alt={{t "loading"}} src={{random-loading-spinner}}>
+      <img alt={{t "loading"}} src=<RandomLoadingSpinner />>
```
Comment (sharpshark28) [inline app/templates/home/chat.hbs]: Umm is this one right? I've never seen syntax quite like this. Worth taking a peek at the page and being sure this works.

Comment (mcfiredrill) [reply app/templates/home/chat.hbs]: Ah shoot, no this is a helper not a component. The codemod didn't seem to recognize the difference. Nice catch. 🎉

**Pull request merged**